src/Core/Entities/ApiKey.cs

```diff
@@ -17,7 +17,7 @@ public class ApiKey : ITableObject<Guid>
     public string EncryptedPayload { get; set; }
     // Key for decrypting `EncryptedPayload`. Encrypted using the organization key.
     public string Key { get; set; }
-    public DateTime ExpireAt { get; internal set; }
```

Comment:
Are we sure its okay to change this?

Reply:
An internal setter restricts the property's setter to only being callable within the same assembly.

So only code in the Core assembly can call it for this specific property.

Since `ExpireAt` is being provided by the client in the HTTP POST request, we need to be able to set it from the Api assembly. 

In the rest of the code base, we are leveraging the internal setter on `CreationDate` and `RevisionDate` properties because those values shouldn't come from the client.

I have a high confidence level that removing the internal is the path forward for this one.

Reply:
I'm going to go ahead and approve the PR, so we can pass the review to Oscar, but we can leave this comment open (a suggestion from Thomas). I think we are safe to remove the `internal` but Oscar can let us know for sure.

Reply:
If we want to keep it as internal we could add it to the constructor. I think that would only make sense if we decide that the expiration date can't be changed using business logic after creating it.